src/features/notes/components/Notes.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import Header from "../../components/Header";
+
+const Notes = () => {
+  return (
+    <div>
+      <Header />
```

Comment:
- Move this component outside, we don't want it imported into every page.

Reply:
Fixed